ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -242,4 +243,99 @@ - (void)testValidateAuthority_whenHostUnreachable_shouldFail
     XCTAssertNil(record);
 }
 
+- (void)testValidateAuthority_whenMultipleAcquireTokenCallsNoMetadataDiffs_shouldOnlyValidateOnce
+{
+    NSString *authority = @"https://login.contoso.com/common";
+    NSString *resource1 = @"resource1";
+    NSString *resource2 = @"resource2";
+    NSUUID *correlationId1 = [NSUUID new];
+    NSUUID *correlationId2 = [NSUUID new];
+    
+    // Network Setup
+    NSArray *metadata = @[ @{ @"preferred_network" : @"login.contoso.com",
+                              @"preferred_cache" : @"login.contoso.com",
+                              @"aliases" : @[ @"sts.contoso.com", @"login.contoso.com"] } ];
+    dispatch_semaphore_t validationSem = dispatch_semaphore_create(0);
+    ADTestURLResponse *validationResponse = [ADTestAuthorityValidationResponse validAuthority:authority withMetadata:metadata];
+    [validationResponse setWaitSemaphore:validationSem];
+    ADTestURLResponse *tokenResponse1 = [self adResponseRefreshToken:TEST_REFRESH_TOKEN
+                                                           authority:@"https://login.contoso.com/common"
```

Comment:
nit: why not use authority variable here instead?

Reply:
👍